gst/nnstreamer/tensor_if/gsttensorif.h

```diff
@@ -35,6 +35,25 @@ G_BEGIN_DECLS
 typedef struct _GstTensorIf GstTensorIf;
 typedef struct _GstTensorIfClass GstTensorIfClass;
 
+/**
+ * @brief Calls the user defined conditions function.
+ * @return TRUE if there is no error
+ */
+typedef gboolean (* tensor_if_custom) (const GstTensorIf * tensor_if,
+    const GstTensorMemory * input, gboolean * result);
+
+/**
+ * @brief Register the custom callback function.
+ */
+extern void
+tensor_if_custom_register (GstElement * tensorif_h, tensor_if_custom func, void *data);
+
+/**
+ * @brief Unregister the custom callback function.
+ */
+extern void
+tensor_if_custom_unregister (GstElement * tensorif_h);
+
```

Comment:
These are nnstreamer APIs (not C-API, yet, though)..

We will need an exportable header for these functions.

Reply:
Thank you for the guide.
I uploaded `nnstreamer_plugin_api_if.h`